packages/@webex/plugin-meetings/test/integration/spec/converged-space-meetings.js

```diff
@@ -1,5 +1,5 @@
-import { config } from 'dotenv';
 import 'jsdom-global/register';
+import { config } from 'dotenv';
```

Comment:
can we fix the formatting here?

Reply:
Wonder why it wasn't fixed automatically.. 
updated, thank you